src/engraving/dom/score.cpp

```diff
@@ -4589,9 +4609,9 @@ ChordRest* Score::findChordRestEndingBeforeTickInStaff(const Fraction& tick, sta
         ptick = m->tick();
     }
 
-    Segment* s      = m->first(SegmentType::ChordRest);
-    track_idx_t strack      = staffIdx * VOICES;
-    track_idx_t etrack      = strack + VOICES;
+    Segment* s = m->first(SegmentType::ChordRest);
+    track_idx_t strack = staffIdx * VOICES;
+    track_idx_t etrack = strack + VOICES;
```

Comment:
If possible, it would be more efficient to use `forceVoice` here:
```c++
```suggestion
    track_idx_t strack = forceVoice ? staffIdx * VOICES + voice : staffIdx * VOICES;
    track_idx_t etrack = forceVoice ? strack + 1 : strack + VOICES;
```
and leave the loop below untouched; this would be equivalent but faster (by avoiding unnecessary work).